test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,812 @@
+          },
+          {
+            'collection': c2,
+            'index': 'inverted'
+          },
+          {
+            'collection': c3_insert,
+            'index': 'inverted'
+          },
+          {
+            'collection': c4_update,
+            'index': 'inverted'
+          },
+          {
+            'collection': c5_replace,
+            'index': 'inverted'
+          },
+          {
+            'collection': c6_not_null,
+            'index': 'inverted'
+          },
+          {
+            'collection': c7_hex,
+            'index': 'inverted'
+          },
+          {
+            'collection': c8_overwriteFalse,
+            'index': 'inverted'
+          },
+          {
+            'collection': c9_overwriteTrue,
+            'index': 'inverted'
+          },
+          {
+            'collection': c10_multiple,
+            'index': 'inverted'
+          },
+          {
+            'collection': c11,
+            'index': 'inverted'
+          },
+          {
+            'collection': c12,
+            'index': 'inverted'
+          }
+        ]
+      })
+
+      //-------------------------------------------------------x-------------------------------------------------------------
+
+      //inserting data to all collection
+      let Collection_array = [a1, a2, a3, a4, a5, a6, a7, a8, a9, a10, a11, a12];
+      let docsAsStr = fs.read(`${PWD}/makedata_suites/060_computed_value.json`);
+
+      // this function will read and insert and check all the neccessary data for the respective collection
+      Collection_array.forEach(col => {
+        col.save(JSON.parse(docsAsStr), { silent: true });
+
+        //this cmd will find one docs from the collection
+        let expected_field = col.all().limit(1).toArray();
+        // print(expected_field);
+        // print(expected_field[0].cv_field)
+        //checking computed value field exit on the collection's doc
+        if (col === a1 || col === a2 || col === a7 || col === a8 || col === a9 || col === a11 || col === a12) {
+          if (expected_field[0].cv_field !== null) {
+            print(`060: ok`)
+          }
+        } else if (col === a3) {
+          if (expected_field[0].cv_field_insert !== null) {
+            print(`060: ok`)
+          }
+        } else if (col === a10) {
+          if (expected_field[0].cv_field1 !== null) {
+            print(`060: ok`)
+          }
+        }else{
+          print(`060: test skipped intentionally!`)
+        }
+      })
+
+      // This method will take input and output array and compare both's results
+      let resultComparision = (input_array, output_array) =>{
+        for(let i=0; i<input_array.length; i++){
+          let output = db._query(input_array[i]).toArray()
+
+          if (Number(output) === output_array[i]) {
+            print("060: ok")
+          } else {
+            throw new Error(`Index query ${output} didn't match with ${output_array[i]}!`);
+          }
```

Comment:
```suggestion
          if (Number(output) !== output_array[i]) {
 throw new Error(`Index query ${output} didn't match with ${output_array[i]}!`);
```

Lets print only necessary messages